src/Toolkit/registry/default/components/Avatar/AvatarFallback.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "AvatarFallback",
+    "manifest": "components/Avatar/AvatarFallback.json",
+    "theme": "",
+    "type": "component",
+    "code": "{%- props -%}\n{%- set style = html_cva(\n    base: 'flex h-full w-full items-center justify-center rounded-full bg-muted',\n    variants: {},\n    compoundVariants: []\n) -%}\n\n<span\n        class=\"{{ style.apply({}, attributes.render('class'))|tailwind_merge }}\"\n        {{ attributes.defaults({}).without('class') }}\n>\n    {% block content %}{% endblock %}\n</span>\n",
```

Comment:
```suggestion
    "template": "{%- props -%}\n{%- set style = html_cva(\n    base: 'flex h-full w-full items-center justify-center rounded-full bg-muted',\n    variants: {},\n    compoundVariants: []\n) -%}\n\n<span\n        class=\"{{ style.apply({}, attributes.render('class'))|tailwind_merge }}\"\n        {{ attributes.defaults({}).without('class') }}\n>\n    {% block content %}{% endblock %}\n</span>\n",
```

Reply:
To allow for different "code" later?

Reply:
I didn't push yet, but this behavior has been improved a lot!
The new ValueObject `Component` has a new property `files`, a list of VO `File` instances (only Twig files for the moment, but can be improved later for StimulusController and more!)